scss/_patterns_notifications.scss

```diff
@@ -90,9 +93,10 @@ $notification-text-margin-bottom: $spv--large - $spv-nudge;
     }
 
     &.is-borderless {
-      background-color: transparent;
+      background-color: $colors--theme--background-default;
```

Comment:
Is this needed?

Borderless notification should "blend in" with it's container, so should have no background (transparent), and no text colour (as it should be inherited from parent).